addons/mail/static/src/models/composer_view.js

```diff
@@ -680,7 +680,7 @@ registerModel({
                 const command = this._getCommandFromText(this.composer.textInputContent);
                 if (command) {
                     await command.execute({ channel: this.composer.thread, body: this.composer.textInputContent });
-                    if (this.composer.exists()) {
+                    if (this.composer != undefined && this.composer.exists()) {
```

Comment:
```suggestion
                    if (this.exists() && this.composer) {
```

When the current record is destroyed, `this.exists()` is falsy. If it's not destroyed, then we can access relational field `.composer` (which might be immediately `falsy` if the related composer no longer exists)